stdlib/2and3/zipfile.pyi

```diff
@@ -141,6 +141,25 @@ class ZipInfo:
         def is_dir(self) -> bool: ...
     def FileHeader(self, zip64: Optional[bool] = ...) -> bytes: ...
 
+if sys.version_info >= (3, 8):
+    class Path:
+        name: str
+        def __init__(self, root: Union[ZipFile, _Path, IO[bytes]], at: _SZI = ...) -> None: ...
+        def open(self, mode: Text = ..., pwd: Optional[bytes] = ...) -> IO[bytes]: ...
+        def iterdir(self) -> Generator[Path, None, None]: ...
```

Comment:
Doesn't look like it's actually a generator. Would replace with `Iterator[Path]`.